windows/CodePush.Shared/CodePushNativeModule.cs

```diff
@@ -240,17 +244,22 @@ public void notifyApplicationReady(IPromise promise)
                 await LoadBundleAsync().ConfigureAwait(false);
             }
         }
-        
+
         internal async Task LoadBundleAsync()
         {
             // #1) Get the private ReactInstanceManager, which is what includes
             //     the logic to reload the current React context.
             FieldInfo info = typeof(ReactPage)
                 .GetField("_reactInstanceManager", BindingFlags.NonPublic | BindingFlags.Instance);
-
+#if WINDOWS_UWP
             var reactInstanceManager = (ReactInstanceManager)typeof(ReactPage)
                 .GetField("_reactInstanceManager", BindingFlags.NonPublic | BindingFlags.Instance)
                 .GetValue(_codePush.MainPage);
```

Comment:
This is more of a question for the CodePush team, but should we just make the fields you need here part of the public API on react-native-windows?